### PR TITLE
Add general guidance, refine single.html

### PR DESCRIPTION
**Pull request by celestehorgan**
Signed-off-by: Celeste Horgan <celeste@cncf.io>

### CODE REVIEW

File: content/general-guidance.md [hunk 1]
```diff
@@ -1,3 +1,127 @@
 ---
 title: "General guidance"
 ---
+
```
Comment (edwarnicke) [inline content/general-guidance.md:4]: This is fantastic... how do we represent the fact that the 'thing' to which the terms being changed are attached may be implemented by/standardized by others and there should be collaboration to ensure everyone is renaming *consistently* ?

Comment (celestehorgan) [reply content/general-guidance.md]: Hmmmmm.

Let me think on this. I think you're bringing up a good point that I need to rephrase for myself.

File: config.toml [hunk 2]
```diff
@@ -47,15 +47,15 @@ name = "Lexend Deca"
...
+url = "/"
+weight = 1
 
-# [[menu.main]]
-# name = "General guidance"
-# url = "/general-guidance"
-# weight = 2
+[[menu.main]]
+name = "Implementation path"
+url = "/Implementation-path"
```
Comment (justaugustus) [inline config.toml:57]:
```suggestion
url = "/implementation-path"
```

?

**Review by justaugustus — CHANGES_REQUESTED** [review]
Question on capitalization of the URL in `config.toml`.

Also, a nitty nit... can we line-wrap at 80 characters?
It'll make future updates easier to review line-by-line.

File: content/implementation-path.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+---
+title: "Implementation Path"
+---
+
+The work to be done in support of naming changes is generally low intensity, but can have consequences. We suggest the following:
```
Comment (xmulligan) [inline content/implementation-path.md:5]: I don't exactly understand what "low intensity" means in this context. Can it be rephrased?

File: content/implementation-path.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+For projects that have automated test suites, add tests to ensure that either both old and new names work, or that at a minimum new names work. This allows end users to incrementally transition with confidence.
+
+In some projects this work will take significant time, be gracious to those doing the work, and be understanding that it will take time to do all this coordination.
+ 
+## Customer-facing changes
+
+Changes to customer facing terminology like URLs are harder to deal with. Run through the steps for internal changes in addition to the following two points.
+
+### URLs, URL fragments, command and API names
```
Comment (xmulligan) [inline content/implementation-path.md:48]:
```suggestion
### URLs, URL fragments, command, and API names
```

File: content/implementation-path.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+Check their external configuration settings, especially all that are not controlled by the version control system for the source code.
+
+Review anything that has branch-specific information, such as protected branch rules
+
+### Changing the git branch name
+
+The best resource for changing branch names is [GitHub's renaming repository](https://github.com/github/renaming).
+
+Their recommendation is to wait until later in the year (2020) to change branch names. However, if you want to start work, we recommend the following:
```
Comment (xmulligan) [inline content/implementation-path.md:88]: it is the end of 2020, has this been updated?

File: content/implementation-path.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+1. Rename branches to `main` in repositories that are not customer facing and are low impact first, for example, documentation repositories.
+2. Next rename repositories with deeper integrations that are outside of the critical release path: adjacent tooling, github organization automation, and smaller projects are good next candidates.
```
Comment (xmulligan) [inline content/implementation-path.md:91]:
```suggestion
2. Next rename repositories with deeper integrations that are outside of the critical release path: adjacent tooling, Github organization automation, and smaller projects are good next candidates.
```

File: content/implementation-path.md (hunk 3 above)
Comment (celestehorgan) [reply content/implementation-path.md:5]: Low effort, not a huge investment of engineering time.. in that realm. I'll try another phrase.

File: content/implementation-path.md (hunk 4 above)
Comment (celestehorgan) [reply content/implementation-path.md:48]: FYI: i'm taking this (and other `suggest` suggestions) but I'm going to add them as a commit via my local and dismiss these as I go :)

File: content/implementation-path.md (hunk 5 above)
Comment (celestehorgan) [reply content/implementation-path.md:88]: It has. GitHub's official line is still end of 2020.
